src/brnd/brnd-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "BRND sportcenter dagsprogram",
+  "icon": "",
+  "id": "01FRJPF4XATRN8PBZ35XN84PS6",
```

Comment:
Important! The id of a new template should be a new ULID. This is the same as calendar template:
https://github.com/os2display/display-templates/blob/develop/src/calendar/calendar-config.json#L4

You can generate a new ULID with the following command (in API):
https://github.com/os2display/display-api-service/blob/develop/src/Command/GenerateUlid.php#L15